src/Clapi/Relay.hs

```diff
@@ -92,6 +93,25 @@ genInitDigest ps tns vs =
         RtnDataSeries ts ->
           d'{ocdData = alInsert p (oppifyTimeSeries ts) $ ocdData d}
 
+contDiff
+  :: AssocList Seg (Maybe Attributee) -> AssocList Seg (Maybe Attributee)
+  -> Map Seg (Maybe Attributee, SequenceOp Seg)
+contDiff a b = afters <> absents
+  where
+    asAfter (acc, prev) k ma = ((k, (ma, SoPresentAfter prev)) : acc, Just k)
+    aftersFor = Map.fromList . fst . alFoldlWithKey asAfter ([], Nothing)
+    aftersA = aftersFor a
+    aftersB = aftersFor b
+    afters = Map.difference aftersB aftersA
+    absents = fmap (const SoAbsent) <$> Map.difference aftersA aftersB
+
+mayContDiff :: Maybe (RoseTreeNode a) -> AssocList Seg (Maybe Attributee) -> Maybe (Map Seg (Maybe Attributee, SequenceOp Seg))
```

Comment:
Long line itus

Reply:
:+1: Yeah, I try for 80 chars for side-by-side diffing ease